internal/jsoncons/CMakeLists.txt

```diff
@@ -14,5 +14,5 @@ FetchContent_Declare(jsoncons
         INSTALL_DIR "${CMAKE_CURRENT_BINARY_DIR}/inst"
         LOG_DIR "${CMAKE_CURRENT_BINARY_DIR}/log"
         )
-
+set(JSONCONS_BUILD_TESTS OFF CACHE BOOL "" FORCE)
```

Comment:
To add context to the conversation earlier in the request, if `CACHE BOOL "" FORCE` wasn't set here, CMake complains about policy CMP0077 not being set (it appears only set in GLFW) and ignores our setting of the option.